components/skus/browser/rs/lib/src/sdk/credentials/mod.rs

```diff
@@ -31,90 +31,80 @@ where
         order_id: &str,
         domain: &str,
     ) -> Result<Option<CredentialSummary>, SkusError> {
-        let order = self.client.get_order(order_id).await?;
-
-        if let Some(order) = order {
-            let expires_at = order.expires_at;
-
-            if !order.location_matches(&self.environment, domain) {
-                return Err(InternalError::OrderLocationMismatch.into());
-            }
+        let wrapped_order = self.client.get_order(order_id).await?;
+        if wrapped_order.is_none() {
+            return Err(InternalError::NotFound.into());
+        }
+        let order = wrapped_order.ok_or(InternalError::NotFound)?;
+        if !order.location_matches(&self.environment, domain) {
+            return Err(InternalError::OrderLocationMismatch.into());
+        }
 
-            // FIXME only returns summary for first item
-            if let Some(item) = order.items.iter().next() {
-                return Ok(match item.credential_type {
-                    CredentialType::SingleUse => self
-                        .client
-                        .get_single_use_item_creds(&item.id)
+        let expires_at = order.expires_at;
+        for item in &order.items {
+            match item.credential_type {
+                CredentialType::SingleUse => {
+                    let wrapped_creds = self.client.get_single_use_item_creds(&item.id).await?;
+                    if let Some(creds) = wrapped_creds {
+                      let unblinded_creds = creds.unblinded_creds.ok_or(InternalError::NotFound)?;
```

Comment:
you've still got some indentation issues, can you try running `cargo fmt` again?

Reply:
done